docs/source/guides/Data-sharing.md

```diff
@@ -0,0 +1,76 @@
+# Data Sharing
+
+## Background
+
+Data is key to neuroscience, and it should be shared for maximal utility. There is increasing pressure from funders and publishers to share data alongside publications:
+
+**Wellcome:**
+
+_“We expect our researchers to maximise the availability of research data, software and materials with as few restrictions as possible. As a minimum, the data underpinning research papers should be made available to other researchers at the time of publication, as well as any original software that is required to view datasets or to replicate analyses.”_
+
+**Cell Press:**
+
+_“All datasets that are composed of standardized datatypes and reported in Cell Press papers must be deposited in a datatype-specific, Cell-Press-recommended repository before a paper is accepted for publication.”_
+
+_“All original code must be either deposited in a repository that mints DOIs or included in the supplemental information before a paper is accepted for publication.”_
+
+
+## Requirements
+
+Currently, SWC does not have any specific code and data sharing requirements, but your funder or publisher may do. 
+
+**Ideally, all data and code will be shared in such a way that any other researcher could easily reproduce key figures from a publication, and build on those data and software in their own research.**
+
+
+## Code
+Original code can be hosted in a public GitHub/GitLab repository, hosted on an organisational account (i.e. not an individual account). This could be the [SWC organisation](https://github.com/SainsburyWellcomeCentre), or that of a particular team, lab or collaboration. Code should include an [appropriate license](https://howto.neuroinformatics.dev/guides/Licensing.html), and may include a [DOI](https://www.doi.org/) (e.g. using [Zenodo](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content)). 
```

Comment:
Personally, I would use stronger language than "may" for recommending attaching persistent identifiers to code repositories on GitHub/GitLab, especially when the code is associated with a manuscript. Given that the repo owner can at any time change or delete the code, without versioned archival on Zenodo or elsewhere, the version of the code associated with a paper is likely to be lost.
That said, we might be overreaching with it. Not everyone needs to hear my strong opinions on this.

Reply:
I'd say it's worth making the distinction between making code public (which is what GitHub does) vs archiving for posterity (what Zenodo does). Doing the first doesn't automatically accomplish the second.

Reply:
I've strengthened it slightly with:
> To ensure a permanent archive of the code associated with a publication a [DOI](https://www.doi.org/) should be created (e.g. using [Zenodo](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content)).

I'm cautious of recommending too many things. If researchers did all the good/code data practices we would like, there may not be any time left to generate data!